.github/workflows/scenario-test.yaml

```
@@ -1,106 +1,94 @@
 name: scenario-test
 on:
+  schedule:
+    - cron: 0 12 * * *
   workflow_dispatch:
-
+  pull_request:
+    paths:
+      - simulator*.repos
+      - .github/workflows/scenario-test.yaml
```

For now, let's only run when there is an update to main branch instead of against pull request.
```suggestion
  push:
    branches:
      - main
```